libs/init_runtime/include/hpx/hpx_init_params.hpp

```diff
@@ -51,6 +51,17 @@ namespace hpx {
     }    // namespace resource
     /// \endcond
 
+    enum runtime_type
+    {
+        runtime_type_local,
+        runtime_type_distributed,
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
+        runtime_type_best = distributed
+#else
+        runtime_type_best = local
+#endif
+    };
+
```

Comment:
For the other enums exposed through the API we have an additional API that can return the string representation of a given enum value. Should we add that here as well? This could be used for generating error messages.

Reply:
That's a good idea. Will add that.